packages/zowe-explorer/src/uss/actions.ts

```diff
@@ -396,7 +458,7 @@ export async function copyUssFiles(node: IZoweUSSTreeNode, nodeList: IZoweUSSTre
     await Gui.withProgress(
         {
             location: vscode.ProgressLocation.Window,
-            title: localize("ZoweUssNode.copyDownload.progress", "Downloading copied files ..."),
+            title: localize("ZoweUssNode.copyDownload.progress", "Copying file structure..."),
```

Comment:
Wasn't sure about this either - without a notification, the copy operation will not provide any feedback to the user. But if that is preferred UX (which I'm thinking is the case because other apps behave in this way), I will remove this.

Reply:
Upon further consideration, in one of the helper functions for copying the structure, we trigger a refresh of USS for files to make sure the paths are up to date. Since this function is `async` and depends on USS to refresh first, I think it will be best to leave this progress bar in here (especially since it is a status bar message and should not be too distracting for the user)